lib/cell.rb

```diff
@@ -0,0 +1,25 @@
+class Cell
+  attr_reader :coordinate, :ship
+  def initialize(coordinate)
+    @coordinate = coordinate
+    @ship = nil
+    @hit = false
```

Comment:
I agree that we need this instance variable to keep track of whether the cell has been fired upon. Light suggestion to change the name from hit to fired_upon. I think hit refers to a ship getting hit whereas a cell can be fired upon even if it's empty. But this is all semantics! I'm okay with whatever you decide.

Reply:
That is a good suggestion. I will see if I can refactor it!

Reply:
I initially tried to change it to fired_upon? but I was getting an error.

`
`attr_reader :coordinate, :ship, :fired_upon?
  def initialize(coordinate)
    @coordinate = coordinate
    @ship = nil
    @fired_upon? = false
  end`

cell.rb:6: syntax error, unexpected '=' (SyntaxError) @fired_upon? = false

Reply:
Found a good stack overflow article explaining why. 

https://stackoverflow.com/questions/14006019/using-a-question-mark-in-ruby-methods

The main point is that I can't use a question mark at the end of variables only methods.

Reply:
I created a fired_upon? method that uses the fired_upon instance variable and it works. I push the change up.